package.json

```diff
@@ -114,16 +114,17 @@
     "@types/lodash": "4.14.195",
     "@types/node": "18.16.16",
     "@types/store": "2.0.2",
-    "@typescript-eslint/eslint-plugin": "^4.33.0",
+    "@typescript-eslint/eslint-plugin": "5.59.8",
+    "@typescript-eslint/typescript-estree": "5.59.8",
```

Comment:
Adding this explicitly is required to avoid our lint scripts defaulting to 4.33 version that is included due to old gatsby required eslint packages.